File: demo-front/src/model/Editor.js
class Editor {
    constructor(id, userId, role) {
      this.role = role;
      this.userId = userId;
      this.id = id;
    }
  
    getRole() {
      return this.role
    }

    getUserId() {
      return this.userId
    }
  
    getId() {
      return this.id
    }
  }
    
  export default Editor;

File: demo-server/src/main/java/com/example/demoserver/model/Editor.java
package com.example.demoserver.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Editor {

    @Id
    private Long id;
    private String userId;
    private String role;
}
